data/CsvDataTool/SiteMap.cs

```diff
@@ -0,0 +1,52 @@
+﻿using CsvHelper.Configuration;
+using Newtonsoft.Json;
+
+namespace CsvDataTool;
+
+public class SiteMap : ClassMap<Site>
+{
+    public SiteMap()
+    {
+        Map(m => m.Id).Name("ID");
+        Map(m => m.Name).Name("Name");
+        Map(m => m.Address).Name("Address");
+        Map(m => m.PhoneNumber).Name("PhoneNumber");
+        Map(m => m.Location).Convert(x =>
+            new Location(
+                "Point",
+                [x.Row.GetField<double>("Longitude"), x.Row.GetField<double>("Latitude")]
+                ));
+        Map(m => m.IntegratedCareBoard).Name("ICB");
+        Map(m => m.Region).Name("Region");
+    }
+}
+
+public record struct Site(
```

Comment:
Can we reference the existing types

Reply:
the CSV library has slightly different requirements for the DTOs, they need default constructors. this was my original approach until i encountered this gotcha in the library